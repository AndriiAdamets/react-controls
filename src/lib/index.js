import Button from './components/Button';
import Card from './components/Card';
import Tooltip from './components/Tooltip';
import Checkbox from './components/Checkbox';
import Input from './components/Input';
export { Button, Card, Tooltip, Checkbox, Input };