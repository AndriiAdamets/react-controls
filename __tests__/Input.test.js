import React from 'react';
import renderer from 'react-test-renderer';
import { Input } from '../src/lib';

test('Input', () => {
  const component = renderer.create(
    <Input />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});