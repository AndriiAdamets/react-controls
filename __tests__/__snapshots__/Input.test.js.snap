// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input 1`] = `
<input
  className="wrc-input"
/>
`;
